resources/aws/elb.go

```diff
@@ -0,0 +1,93 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elb"
+	awsclient "github.com/giantswarm/aws-operator/client/aws"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+// ELB is an Elastic Load Balancer
+type ELB struct {
+	Name          string
+	AZ            string
+	SecurityGroup string
+	Tags          []string
+	Client        *elb.ELB
+}
+
+func (lb *ELB) CreateIfNotExists() (bool, error) {
+	if lb.Client == nil {
+		return false, microerror.MaskAny(clientNotInitializedError)
+	}
+
+	if err := lb.CreateOrFail(); err != nil {
+		if err.Error() == awsclient.ELBAlreadyExists {
+			return false, nil
+		}
+
+		return false, microerror.MaskAny(err)
+	}
+
+	return true, nil
+}
+
+func (lb *ELB) CreateOrFail() error {
+	if lb.Client == nil {
+		return microerror.MaskAny(clientNotInitializedError)
+	}
```

Comment:
Really specific nitpick, a new line here to match the above function would be cool